config-model/src/test/java/com/yahoo/vespa/model/container/search/test/empty.cfg

```diff
@@ -0,0 +1 @@
+enableGroupingSessionCache false
```

Comment:
Is this file necessary? If so why, and why the name "empty.cfg"?

Sigh, these cfg files in tests are a really bad idea, but not everyone agrees.

Reply:
I have no idea 🙈 I agree, seems overkill to check in an empty file for this kind of assertion.